### PR TITLE
documentation-description

### PR DESCRIPTION
**Pull request by mariaathena**
# description

add description and usage example to documentation

### CODE REVIEW

**Comment by mariaathena** [comment]
will be fixed with #2

File: README.md [hunk 1]
```diff
@@ -1,10 +1,31 @@
 # purple-pluto
 
-Todo: add description
+## Description
+
+Tool for tracking visitor activity on your web application. Deploying the code from this repository will allow you to record and store data of (x,y) coordinates from visitor's following actions:
+
+- Mouse hoover
```
Comment (mattiaerre) [inline README.md]: although I think this would be a really cool event to track, I believe there is a typo here 😅

File: README.md [hunk 2]
```diff
@@ -1,10 +1,31 @@
...
+- Mouse clicks
+- Scroll on page
```
Comment (mattiaerre) [inline README.md]: clicks and scrolling events are not implemented yet, so I would only mention `mousemove` events for now

File: README.md [hunk 3]
```diff
@@ -1,10 +1,31 @@
...
 
 ## Usage
 
-Todo: add usage description
+Todo: improve usage description
```
Comment (mattiaerre) [inline README.md]: probably better to leave this section as a todo/wip for now

File: README.md [hunk 4]
```diff
@@ -14,6 +35,11 @@ Todo: add methods description
 
 Todo: add config description
 
+## Requirements
```
Comment (mattiaerre) [inline README.md:22]: I like this `Requirements` section, let's expand it a bit

**Review by mattiaerre — CHANGES_REQUESTED** [review]
over to you @mariaathena please have a look at my notes 😉

File: README.md [hunk 5]
```diff
@@ -1,10 +1,15 @@
...
+
+Tool for tracking visitor activity on your web application. Deploying the code from this repository will allow you to record and store data of (x,y) coordinates from visitor's following actions:
+
+- Mouse hover
+- (more to follow)
 
 ## Usage
 
-Todo: add usage description
+Todo: add usage description`
```
Comment (mattiaerre) [inline README.md]: You missed a `'` 😅

Comment (mattiaerre) [reply README.md]: @mariaathena if you fix this I will merge asap 👍

Comment (mariaathena) [reply README.md]: my bad, all done!

**Review by mattiaerre — APPROVED** [review]
this LGTM

**Pull request merged**